packages/terra-aggregator/src/terra-dev-site/doc/aggregator/Aggregator.1.doc.mdx

```diff
@@ -71,74 +83,20 @@ By default, the components will be rendered by the Aggregator in the order given
 ### Example
 
 ```jsx
-import React from 'react';
-import Base from 'terra-base';
-import Button from 'terra-button';
 import Aggregator from '../../src/Aggregator';
```

Comment:
We'll want to either remove this example or switch this import example to pull from the released component. 

```jsx
import Aggregator from 'terra-aggregator';
```

Reply:
Right now it renders like this:

![image](https://user-images.githubusercontent.com/6720991/73001665-742bc300-3dc8-11ea-97da-7a5fd287056b.png)

Reply:
Changed in 8edd522